src/PortlandLabs/ConcreteCmsTheme/Navigation/HeaderNavigationFactory.php

```diff
@@ -23,33 +23,65 @@ class HeaderNavigationFactory implements ApplicationAwareInterface, NavigationFa
     public function createNavigation(): NavigationInterface
     {
         $activeSection = $this->activeSection;
+        // Determine if this navigation menu is for the concretecms.org site or not
+        $siteTarget = $_ENV['URL_SITE_MARKETING_OPENSOURCE'];
+        $siteDetected = \Core::make('site')->getSite()->getSiteCanonicalURL();
+        // remove trailing slash if exists
+        if (strcmp('/', substr($siteDetected, -1)) === 0) {
+            $siteDetected = substr($siteDetected, 0, -1);
+        }
+        $siteIsMarketingOpensource = false;
+        if (strcmp($siteTarget, $siteDetected) === 0) {
```

Comment:
`strcmp` is unnecessary here. PHP comparison operators are generally binary safe so `$siteTarget === $siteDetected` is effectively equivalent and also easier to read at a glance.